lib/issues.go

```diff
@@ -37,15 +37,18 @@ func CompareIssues(config cfg.Config, ghClient clients.GitHubClient, jiraClient
 		ids[i] = v.GetID()
 	}
 
-	jiraIssues, err := jiraClient.ListIssues(ids)
-	if err != nil {
-		return err
-	}
-
 	log.Debug("Collected all JIRA issues")
 
 	for _, ghIssue := range ghIssues {
 		found := false
+
+		ids := make([]int, 1)
```

Comment:
We could same some time by allocating this slice only once, before the loop, and then reusing it.